templates/package/content/swift.tmpl

```diff
@@ -0,0 +1,40 @@
+{{if eq .PackageDescriptor.Package.Type "swift"}}
+	<h4 class="ui top attached header">{{.locale.Tr "packages.installation"}}</h4>
+	<div class="ui attached segment">
+		<div class="ui form">
+			<div class="field">
+				<label>{{svg "octicon-terminal"}} {{.locale.Tr "packages.swift.registry"}}</label>
+				<div class="markup"><pre class="code-block"><code>swift package-registry set {{AppUrl}}api/packages/{{.PackageDescriptor.Owner.Name}}/swift</code></pre></div>
```

Comment:
The `AppUrl` could be replaced by relative URLs
* #22861

Reply:
This is a command line argument for swift. I don't think it can guess the domain you want to use 😄

Reply:
> This is a command line argument for swift. I don't think it can guess the domain you want to use 😄

I think we can. If he can visit the current page, we can give him the current host but not the fixed domain.